asserts/database.go

```diff
@@ -209,14 +185,13 @@ func (db *Database) findAccountKeys(authorityID, keyID string) ([]*AccountKey, e
 		}
 	}
 	// consider stored account keys
-	foundKeyCb := func(a Assertion) {
+	a, err := db.bs.Get(AccountKeyType, []string{"account-id", "public-key-id"}, []string{authorityID, keyID})
```

Comment:
We shouldn't be duplicating knowledge about the assertion types like this.

Can we please define something like:
```
func primaryKey(t AssertionType) []string {
        if typeData, ok := typeRegistry[t]
        if !ok {
                panic("unknown assertion type used as value: " + string(t))
        }
        return typeData.primaryKey
}
```

and then use it as

    db.bs.Get(AccountKeyType, primaryKey(AccountKeyType), []string{...})

At some point perhaps assertion types should be richer, so we don't need to be even passing that information around independently like that.